core/state_transition.go

```diff
@@ -748,6 +765,24 @@ func (st *stateTransition) refundGas(refundQuotient uint64) uint64 {
 	return refund
 }
 
+func (st *stateTransition) refundOperatorCost() {
+	if optimismConfig := st.evm.ChainConfig().Optimism; optimismConfig != nil && !st.msg.IsDepositTx && st.evm.ChainConfig().IsOptimismIsthmus(st.evm.Context.Time) {
```

Comment:
check can be removed if flanking `refundOperatorCost()` call with `if rules.IsOptimismIsthmus {` at the calling site.

Reply:
Done